README.md

```diff
@@ -1,4 +1,6 @@
-# Cloudflow
+<p>
+<img src="./docs/images/logo.svg" width="300">
```

Comment:
maybe there is better way than hardcoded values, but it seems to work (needs check).